apps/backend/src/authn/authn.controller.ts

```diff
@@ -126,8 +130,12 @@ export class AuthnController {
       accessToken: string;
     }
   ): Promise<void> {
-    req.res?.cookie('userID', session.userID);
-    req.res?.cookie('accessToken', session.accessToken);
+    req.res?.cookie('userID', session.userID, {
+      secure: this.configService.get('NODE_ENV')?.toLowerCase() === 'production'
+    });
+    req.res?.cookie('accessToken', session.accessToken, {
+      secure: this.configService.get('NODE_ENV')?.toLowerCase() === 'production'
+    });
```

Comment:
It would be cleaner if the check for production was made within the configService itself, similar to 

```
  isRegistrationAllowed(): boolean {
    return this.get('REGISTRATION_DISABLED')?.toLowerCase() !== 'true';
  }
```